src/main/java/com/aetherteam/aether_genesis/item/combat/PhoenixDartShooterItem.java

```diff
@@ -0,0 +1,162 @@
+package com.aetherteam.aether_genesis.item.combat;
+
+import com.aetherteam.aether.client.AetherSoundEvents;
+import com.aetherteam.aether.entity.projectile.dart.AbstractDart;
+import com.aetherteam.aether.item.combat.DartItem;
+import com.aetherteam.aether_genesis.GenesisTags;
+import com.aetherteam.aether_genesis.entity.PhoenixDart;
+import com.google.common.collect.ImmutableSet;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.stats.Stats;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResultHolder;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.entity.projectile.AbstractArrow;
+import net.minecraft.world.item.*;
+import net.minecraft.world.item.enchantment.Enchantment;
+import net.minecraft.world.item.enchantment.Enchantments;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.event.ForgeEventFactory;
+
+import java.util.function.Predicate;
+
+public class PhoenixDartShooterItem extends ProjectileWeaponItem implements Vanishable {
```

Comment:
I believe you should be able to just extend DartShooterItem. You can use customDart() to create the Phoenix Dart projectile, override getAllSupportedProjectiles to check the dart tag for usable ammo for the dart shooter, and I guess use golden darts as a default dart type parameter.